### PR TITLE
Add icons for pos dropdown

### PR DESCRIPTION
**Pull request by Crystalwarrior**
image

### CODE REVIEW

File: src/courtroom.cpp [hunk 1]
```diff
@@ -3752,7 +3757,7 @@ void Courtroom::play_sfx()
         ao_app->get_sfx_looping(current_char, current_emote) == "1");
 }
 
-void Courtroom::set_scene(QString f_desk_mod, QString f_side)
+QString Courtroom::get_pos_path(QString pos, bool desk)
```
Comment (oldmud0) [inline src/courtroom.cpp:3760]: this should be moved to the path functions file because it is a path function

File: include/courtroom.h [hunk 2]
```diff
@@ -184,6 +184,9 @@ class Courtroom : public QMainWindow {
   // Sets the local pair list based on the characters available on the server.
   void set_pair_list();
 
+  // get pos filepath for bg or desk
+  QString get_pos_path(QString pos, bool desk = false);
```
Comment (stonedDiscord) [inline include/courtroom.h:188]: sounds like a path function to me

**Review by stonedDiscord — DISMISSED** [review]
cute!!

**Review by oldmud0 — DISMISSED** [review]
LGTM though i'm finding myself squinting at those little icons

**Comment by stonedDiscord** [comment]
i think the bot is right and these variables should be const

i might do this tomorrow

**Review by stonedDiscord — APPROVED** [review]

**Pull request merged**